js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js

```diff
@@ -76,7 +76,7 @@
     createIndex: function () {
       // e.preventDefault()
       var self = this;
-      var indexType = $('#newIndexType').val();
+      var indexType = arangoHelper.escapeHtml($('#newIndexType').val());
```

Comment:
escaping should not be done here.

Reply:
please only escape when displaying a value, not when reading it or working with it internally.
but please make sure that whenever a value is displayed, it has been escaped.